app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/UpscanService.scala

```diff
@@ -51,41 +47,15 @@ trait UpscanService {
     implicit hc: HeaderCarrier
   ): EitherT[Future, Error, Unit]
 
-  def removeFile(draftReturnId: DraftReturnId, upscanInitiateReference: UpscanInitiateReference)(
-    implicit hc: HeaderCarrier
-  ): EitherT[Future, Error, Unit]
-
-  def removeAllFiles(draftReturnId: DraftReturnId)(
-    implicit hc: HeaderCarrier
-  ): EitherT[Future, Error, Unit]
-
-  def getAll(draftReturnId: DraftReturnId)(
-    implicit hc: HeaderCarrier
-  ): EitherT[Future, Error, List[UpscanFileDescriptor]]
-
 }
 
 @Singleton
 class UpscanServiceImpl @Inject() (
-  upscanConnector: UpscanConnector,
-  configuration: Configuration
-) extends UpscanService
+  upscanConnector: UpscanConnector
+)(implicit ec: ExecutionContext)
+    extends UpscanService
     with Logging {
 
-  private def getUpscanInitiateConfig[A: Configs](key: String): A =
-    configuration.underlying
-      .get[A](s"microservice.services.upscan-initiate.$key")
-      .value
-
-  private val maxUploads: Int = getUpscanInitiateConfig[Int]("max-uploads")
-
-  private def hasReachedMaxFileUpload(upscanSnapshot: UpscanSnapshot): Either[Error, Unit] =
-    if (upscanSnapshot.fileUploadCount >= maxUploads) {
-      Left(Error(MaxUploads))
-    } else {
-      Right(())
-    }
-
   private def augmentUpscanInitiateResponse(
     draftReturnId: DraftReturnId,
     cgtReference: CgtReference,
```

Comment:
I think the cgt reference is redundant in the whole scheme of things but we can address that debt later.